FAILED `test.py::test_notebook[automl_timeseries_forecasting_with_pycaret.ipynb]`

## Problem
```
=========================== short test summary info ============================
FAILED test.py::test_notebook[automl_timeseries_forecasting_with_pycaret.ipynb] - nbclient.exceptions.CellTimeoutError: A cell timed out while it was being executed, after 60 seconds.
The message was: Cell execution timed out.
Here is a preview of the cell contents:
-------------------
['target_data = pd.read_csv(', '    "https://data.4tu.nl/file/539debdb-a325-412d-b024-593f70cba15b/a801f5d4-5dfe-412a-ace2-a64f93ad0010"', ')', 'related_data = pd.read_csv(', '    "https://data.4tu.nl/file/539debdb-a325-412d-b024-593f70cba15b/f2bd27bd-deeb-4933-bed7-29325ee05c2e",']
```
-- https://github.com/crate/cratedb-examples/actions/runs/12819074099/job/35745960097?pr=811#step:6:1626

## Evaluation
It looks like loading times of those resources are currently high, so executing the notebook which acquires them runs into a timeout error.
- https://data.4tu.nl/file/539debdb-a325-412d-b024-593f70cba15b/a801f5d4-5dfe-412a-ace2-a64f93ad0010
- https://data.4tu.nl/file/539debdb-a325-412d-b024-593f70cba15b/f2bd27bd-deeb-4933-bed7-29325ee05c2e

## Problem
Just enumerating/sampling them, not investigating them at all.
```
HTTPError: HTTP Error 502: Bad Gateway
FAILED test.py::test_file[automl_timeseries_forecasting_with_pycaret.py] - urllib.error.HTTPError: HTTP Error 502: Bad Gateway
```
-- https://github.com/crate/cratedb-examples/actions/runs/12860637559/job/35852836619?pr=816#step:6:1677

## Problem
Once in a while, we also receive those errors on CI validation runs against `automl_timeseries_forecasting_with_pycaret.py`:
```python
 >       if not isinstance(obj, dask.dataframe.core.DataFrame):
E       AttributeError: module 'dask.dataframe.core' has no attribute 'DataFrame'

/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/sktime/datatypes/_adapter/dask_to_pd.py:137: AttributeError
```
-- https://github.com/crate/cratedb-examples/actions/runs/12860637559/job/35894717404?pr=816#step:6:1443

Inside here, there is also this to be observed, in this case also flagged as an ERROR:
```python
ERROR    traitlets:__init__.py:98 Notebook JSON is invalid: Additional properties are not allowed ('metadata' was unexpected)

Failed validating 'additionalProperties' in stream:

On instance['cells'][22]['outputs'][3]:
{'metadata': {'nbreg': {'diff_ignore': ['/outputs']}},
 'name': 'stdout',
 'output_type': 'stream',
 'text': 'Fitting 3 folds for each of 10 candidates, totalling 30 fits\n'}
WARNING  traitlets:client.py:1234 No handler found for comm target 'dash'
```
-- https://github.com/crate/cratedb-examples/actions/runs/12860637559/job/35894717404?pr=816#step:6:1368

--- a/topic/machine-learning/pycaret/automl_timeseries_forecasting_with_pycaret.ipynb
+++ b/topic/machine-learning/pycaret/automl_timeseries_forecasting_with_pycaret.ipynb
@@ -230,10 +230,10 @@
    "outputs": [],
    "source": [
     "target_data = pd.read_csv(\n",
-    "    \"https://data.4tu.nl/file/539debdb-a325-412d-b024-593f70cba15b/a801f5d4-5dfe-412a-ace2-a64f93ad0010\"\n",
+    "    \"https://cdn.crate.io/downloads/datasets/cratedb-datasets/machine-learning/timeseries/4tu-a801f5d4-5dfe-412a-ace2-a64f93ad0010.csv\"\n",
     ")\n",
     "related_data = pd.read_csv(\n",
-    "    \"https://data.4tu.nl/file/539debdb-a325-412d-b024-593f70cba15b/f2bd27bd-deeb-4933-bed7-29325ee05c2e\",\n",
+    "    \"https://cdn.crate.io/downloads/datasets/cratedb-datasets/machine-learning/timeseries/4tu-f2bd27bd-deeb-4933-bed7-29325ee05c2e.csv\",\n",
     "    header=None,\n",
     ")\n",
     "related_data.columns = [\"item\", \"org\", \"date\", \"unit_price\"]\n",

--- a/topic/machine-learning/pycaret/automl_timeseries_forecasting_with_pycaret.py
+++ b/topic/machine-learning/pycaret/automl_timeseries_forecasting_with_pycaret.py
@@ -34,10 +34,10 @@ os.environ["MLFLOW_TRACKING_URI"] = DBURI_MLFLOW
 
 def prepare_data():
     target_data = pd.read_csv(
-        "https://data.4tu.nl/file/539debdb-a325-412d-b024-593f70cba15b/a801f5d4-5dfe-412a-ace2-a64f93ad0010"
+        "https://cdn.crate.io/downloads/datasets/cratedb-datasets/machine-learning/timeseries/4tu-a801f5d4-5dfe-412a-ace2-a64f93ad0010.csv"
     )
     related_data = pd.read_csv(
-        "https://data.4tu.nl/file/539debdb-a325-412d-b024-593f70cba15b/f2bd27bd-deeb-4933-bed7-29325ee05c2e",
+        "https://cdn.crate.io/downloads/datasets/cratedb-datasets/machine-learning/timeseries/4tu-f2bd27bd-deeb-4933-bed7-29325ee05c2e.csv",
         header=None,
     )
     related_data.columns = ["item", "org", "date", "unit_price"]

